### DISCUSSION TITLE
Time to start the backend

### DISCUSSION THREAD

**Discussion by abdinasir-Tman** [dugsiiyeinc/dhaqsoPay#8] [dhaqsoPay Server Developers]
@dugsiiyeinc/dhaqsopay-server-developers       engineer sabtiga ayaan bilaabaynaa inshaa allah

**Comment by sharafdin** [comment]
> @dugsiiyeinc/dhaqsopay-server-developers engineer sabtiga ayaan bilaabaynaa inshaa allah

noo kala tuura Insha'Allah!

**Comment by abdinasir-Tman** [comment]
diyaar waaye bro one day inaga badan

**Reply by apdyqafaar** [reply]
ok bro ia
